TYPO3.Neos/Resources/Public/JavaScript/Content/Navigate/NodeTree.js

```diff
@@ -276,6 +276,13 @@ define(
 				if (isCurrentNode) {
 					ContentModule.loadPage(node.data.href);
 				}
+
+				// if the current viewed page is a children of the moved page load it
+				var explodedPath = node.data.key.split('@');
+				if (this.get('pageNodePath').indexOf(explodedPath[0]) === 0) {
+					ContentModule.loadPage(node.data.href);
+				}
```

Comment:
this needs to be an else statement, since it will also match for the current node so it will load the page twice if you move the node you're currently viewing (tested it)